omegaconf/nodes.py

```diff
@@ -87,11 +82,9 @@ def _deepcopy_impl(self, res: Any, memo: Dict[int, Any]) -> None:
 
     def _is_none(self) -> bool:
         if self._is_interpolation():
-            node = self._dereference_node(
-                throw_on_resolution_failure=False, throw_on_missing=False
-            )
+            node = self._dereference_node(throw_on_resolution_failure=False)
```

Comment:
shall we fix is_none to not resolve as well here or is this planned for a followup?

Reply:
See #547